templates.go

```diff
@@ -914,12 +914,6 @@ coreos:
     content: |
       {{range .Content}}{{.}}
       {{end}}{{end}}
-  - name: sshd.service
-    enable: true
-  - name: sshd.socket
-    enable: false
-    mask: true
```

Comment:
should we maybe leave the masking or is that also problematic

Reply:
yeah, its also problematic